gcp/appengine/frontend3/src/templates/vulnerability.html

```diff
@@ -163,57 +200,57 @@ <h3 class="mdc-layout-grid__cell--span-3">
               </dd>
             </dl>
           {% endfor -%}
+          </div>
```

Comment:
We also have per-range ecosystem/database_specific: https://ossf.github.io/osv-schema/#affectedecosystem_specific-field

Reply:
To clarify, in total there's 3 variations here:

- affected[].ranges[].database_specific
- affected[].database_specific field
- affected[].ecosystem_specific field

Reply:
Added a commit to expose it: [47f2185](https://github.com/google/osv.dev/pull/2132/commits/47f218552199cc638d35f041925e74c8cdac915a)

Also looking at the data models, I reckon currently this data is not recorded on the `Bug` model: https://github.com/google/osv.dev/blob/47f218552199cc638d35f041925e74c8cdac915a/osv/models.py#L185-L194
(Please correct me if I'm missing something.)

It might be worth mentioning, I couldn't find any bug that include `database_specific` on their `ranges` though.

sample output:

<img width="1256" alt="Screenshot 2024-05-15 at 12 54 52 PM" src="https://github.com/google/osv.dev/assets/73332835/943b4b71-fae9-488b-bc93-a90a0518e312">

Reply:
@oliverchang - FYI added [ecae833](https://github.com/google/osv.dev/commit/ecae8332f0207be21973935208e0f0cc13bd7e93) to render the on the fly `database_specific` data too.